cli.js

```diff
@@ -47,7 +47,7 @@ if (argv.c) {
 }
 
 function validateCmdVariable (param) {
-  if (!param.match(/^\w+=[a-zA-Z0-9"=^!?%@_&\-/:;.]+$/)) {
+  if (!param.match(/^\w+=[a-zA-Z0-9"=^!?%@_&\-/:;.*]+$/)) {
```

Comment:
```suggestion
  if (!param.match(/^\w+=[a-zA-Z0-9"=^!?%@_&\-/:;.*\\]+$/)) {
```

Is it possible to add backslash?